CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # libhoney Changelog
 
+## 1.5.2
```

Comment:
We've not been consistent across all repos, but I think it could be helpful to include the date of the release like we do on most of them.

```suggestion
## 1.5.2 2022-08-09
```